lib/sass.dart

```diff
@@ -105,7 +107,9 @@ CompileResult compileToResult(String path,
         bool quietDeps = false,
         bool verbose = false,
         bool sourceMap = false,
-        bool charset = true}) =>
+        bool charset = true,
+        Set<Deprecation> fatalDeprecations = const {},
+        Set<Deprecation> futureDeprecations = const {}}) =>
```

Comment:
Non-boolean parameters should generally have `null` defaults, so that we can change default behavior later without changing the public signature.

Reply:
Done